src/component/DashboardTable/OrganizationGrid.tsx

```diff
@@ -0,0 +1,374 @@
+import { Close, Delete, Edit } from "@mui/icons-material";
+import SearchIcon from "@mui/icons-material/Search";
+import {
+	Box,
+	Button,
+	ButtonGroup,
+	IconButton,
+	InputBase,
+	Modal,
+	Pagination,
+	Table,
+	TableBody,
+	TableCell,
+	TableHead,
+	TableRow,
+	TextField,
+	Toolbar,
+	Typography,
+	styled,
+} from "@mui/material";
+import React, { useEffect, useState } from "react";
+import Title from "./Title";
+import axios from "axios";
+
+interface RowData {
```

Comment:
Each interface should be defined in a single file, and imported anywhere when needed. You can create a folder to store all your interfaces